packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -73,16 +73,17 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
               'By default, all build configurations are built.'
       )
       ..addFlag('universal',
-        help: 'Produce universal frameworks that include all valid architectures. '
-              'This is true by default.',
-        defaultsTo: true,
```

Comment:
This should have a https://flutter.dev/docs/release/breaking-changes page if we're changing the default perhaps? Is there any user actions possibly needed for the integrator?

Reply:
We don't usually do that for tool changes, though the integrator would need to pass in `--universal --no-xcframework` now to get the old default behavior.
@zanderso what do you think?

Reply:
As long as the guidance from the tool itself is clear about what is going on, then that is usually sufficient.